lib/converters/converter-v2-to-v1.js

```diff
@@ -354,7 +354,7 @@ _.extend(Builders.prototype, {
         });
 
         // description transformations for v2 to v1
-        urlObj && (request.pathVariableData = urlObj.variables || urlObj.variable);
+        urlObj && (request.pathVariableData = urlObj.variable);
```

Comment:
This change was unnecessary, but good it showed up in the diff. When converting from v2 to v1, you need to again change `id` to `key`.